site/_docs/pagination.md

```diff
@@ -133,8 +133,8 @@ attributes:
 <div class="note info">
   <h5>Pagination does not support tags or categories</h5>
   <p>Pagination pages through every post in the <code>posts</code>
-  variable regardless of variables defined in the YAML Front Matter of
-  each. It does not currently allow paging over groups of posts linked
+  variable unless a post has `hidden: true` in its YAML Front Matter.
```

Comment:
This should be `<code>hidden: true</code>` because the markdown parser doesn't parse inside HTML blocks. 😄